frontend/lib/norent/tests/__snapshots__/letter-email-to-user.test.tsx.snap

```diff
@@ -15,7 +15,9 @@ exports[`NorentLetterEmailToUser works 1`] = `
     You've sent your NoRent letter. Attached to this email is a PDF copy for your records.
   </p>
   <p>
-    To learn more about what to do next, check out our FAQ page: https://myserver.com/en/faqs
+    You can also track the delivery of your letter using USPS Tracking:
+     
+    https://tools.usps.com/go/TrackConfirmAction?tLabels=[object Object]
```

Comment:
Hmm, why does this snapshot's URL end with `tLabels=[object Object]` instead of an actual fake tracking number?  I wonder what the value of `session.norentLatestLetter.trackingNumber` is in the context of the test...

Reply:
Soooo, I spent some time investigating this. Even after I added a trackingNumber to the test session, the error still kept coming up. I was able to fix the issue by removing the `TrackingInfo` component entirely, that received the tracking number as a prop from the parent component (which by the way ended up being a cleaner, more readable way to format the code). I'm still not sure what the problem was but I'm 99% sure it had to do with the passing of the `trackingNumber` prop like that. For some reason, once accessed from the child component, it was read as an object instead of a string/undefined... Anyway the problem seems to be fixed now, snapshot updated and all!

Reply:
As a sanity check, I also removed `session.norentLatestLetter.trackingNumber` is in the context of the test and recreated the snapshot to make sure that the whole tracking info section gets removed, and it does!